server/coordinator.go

```diff
@@ -520,3 +563,8 @@ func (s *scheduleController) GetInterval() time.Duration {
 func (s *scheduleController) AllowSchedule() bool {
 	return s.Scheduler.IsScheduleAllowed(s.cluster)
 }
+
+// isPaused returns if a schedueler is paused.
+func (s *scheduleController) isPaused() bool {
+	return time.Now().UTC().Unix() < s.DelayUntil
```

Comment:
`UTC()` is not necessary as well. There is no difference with or without `UTC()`